osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmap.cs

```diff
@@ -288,6 +295,9 @@ public MenuItem[] ContextMenuItems
 
                 items.Add(new OsuMenuItem("Collections") { Items = collectionItems });
 
+                if (beatmapInfo.GetOnlineURL(api) is string url)
```

Comment:
No ruleset spec here? There is one on the beatmap set panels. And it does matter for stuff like converts.

Reply:
Yeah right, this was on my todo and I forgot to act on it.